Revit_Engine/Query/Plane.cs

```diff
@@ -34,6 +34,7 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
+        [Deprecated("4.0", "BH.Engine.Adapters.Revit.Query.Plane method is not used any more.")]
```

Comment:
The deprecated attribute is actually... Deprecated.

![image](https://user-images.githubusercontent.com/22005920/94652517-74d30c80-02fa-11eb-88e8-697be429f637.png)


If this is to simply be deleted, should be using the `ToBeRemovedAttribute` instead, if replaced by something else, versioning should generally be the first goto, if not at all possible, the `ReplacedAttribute` should be used.

Reply:
Depr... fixed ;-)